src/game/bundles/entity.rs

```diff
@@ -185,6 +188,56 @@ impl MonsterBundle {
     }
 }
 
+#[derive(Bundle)]
+pub struct DroppedItemBundle {
```

Comment:
Nice, we should update the item dropping on death in the systems/npc_ai.rs to use this bundle. We can do that separately from this PR if you want though.

Reply:
Yeah the plan is to do it in a separate PR.